### ISSUE TITLE
Symfony functional testing

### ISSUE THREAD

**Issue by vitalyiegorov** [paratestphp/paratest#330]
Hi @andreasschroth, 

I am trying to use paratest for our API functional tests but got:
```
Symfony\Component\DependencyInjection\Exception\ServiceNotFoundException: You have requested a non-existent service "test.client".
```

Can you please help me with this issue? I believe problem is related to booting kernel/container because with simple `vendor/bin/phpunit` our 400+ tests are green.

**Comment by vitalyiegorov** [comment]
Further investigation is showing that parapets somehow avoids symfony .env loading, thats is why its not seeing `test.client` as it does not uses `<php> <env ...` section from `phpunit.xml` which is setting environment variables.

**Comment by wimpog** [comment]
@vitalyiegorov could you please explain how to fix it? I'm having the same issue.

**Comment by vitalyiegorov** [comment]
@wimpog I have added my own `bin/paratest`:
```php
#!/usr/bin/env php
<?php

use ParaTest\Console\ParaTestApplication;
use Symfony\Component\Dotenv\Dotenv;

$files = array(
    __DIR__ . '/../../vendor/autoload.php',
    __DIR__ . '/../../../autoload.php',
    dirname(__DIR__) . '/vendor/autoload.php'
);

$found = false;

foreach ($files as $file) {
    if (file_exists($file)) {
        require $file;
        $found = true;
        break;
    }
}

if (!$found) {
    die(
        'You need to set up the project dependencies using the following commands:' . PHP_EOL .
        'curl -s http://getcomposer.org/installer | php' . PHP_EOL .
        'php composer.phar install' . PHP_EOL
    );
}

#use ParaTest\UI\Text\PHPUnitTextUI;

#PHPUnitTextUI::main();

$_SERVER['APP_ENV'] = 'test';

$app = new ParaTestApplication();
$app->run();
```
and added it to my symfony bin folder.

**Comment by johnnybenson** [comment]
It looks like this commit https://github.com/paratestphp/paratest/commit/0c48af96fd9a5fe36dd5f14282ac5dc5939f109c has resolved this issue with Symfony + ENV

Try upgrading to 1.1.0!

**Comment by andreasschroth** [comment]
I believe, this should be fixed by PR #318, which just got merged.
